src/index.css

```diff
@@ -1,3 +1,7 @@
 :root {
   background-image: repeating-linear-gradient(0deg, transparent, transparent 1px, rgba(0, 0, 0, 0.02) 1px, rgba(0, 0, 0, 0.02) 2px);
 }
+
+* {
+  padding: 0;
+}
```

Comment:
No end of line.
Again, when you will have prettier on save, these cases will not happen, so I recommend again :)